src/program/lwaftr/tests/propbased/prop_sameval.lua

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env luajit
```

Comment:
Nice property. WDYT about occasionally hammering it with way more than 2 gets/sets, like 100-1000, maybe 1 time in 100?